integration-tests/emulated/penpal/Cargo.toml

```diff
@@ -0,0 +1,193 @@
+[package]
+name = "penpal-runtime-dotsama"
+version = "0.14.0"
+authors = ["Anonymous"]
+description = "TODO: remove `penpal-runtime-dotsama` with 1.3.0 (https://github.com/polkadot-fellows/runtimes/issues/186) - A parachain for communication back and forth with XCM of assets and uniques."
```

Comment:
@claravanstaden 
you can remove this whole module now, and use patched `penpal-runtime@0.15.1` with `CustomizableAssetFromSystemAssetHub` as we did for roroco with `set_storage`

Reply:
Done. 😄